src/components/timeline/TimelineCanvas.js

```diff
@@ -178,33 +187,3 @@ export default class TimelineCanvas<HoveredItem> extends PureComponent<
     );
   }
 }
-
-/**
- * Check for shallow equality for objects, and strict equality for everything else.
- */
-function hoveredItemsAreEqual(a: any, b: any) {
```

Comment:
Ah yeah, I see this in the flame chart only. I'll look at it, maybe it's still needed after all.